compiler/AST/AstDump.cpp

```diff
@@ -182,6 +182,13 @@ bool AstDump::enterDefExpr(DefExpr* node) {
   bool    retval = true;
 
   if (isModuleSymbol(sym)) {
+    newline();
+    write("def");
+    write("module");
+    write(sym->name);
+    if (fLogIds)
+      fprintf(mFP, "[%d]", sym->id);
+    write(" ");
```

Comment:
Just for the sake of my understanding:  Is there anything about this that's specific to included submodules, or is it code we should've had for (sub)modules all along and simply didn't?

Reply:
> is it code we should've had for (sub)modules all along and simply didn't?

I feel this way about it but not all developers view the logs in the same way...

Reply:
Was the previous result that submodules were dropped on the floor, or were they emitted to their own file with no mention of the relation to the parent module in the parent module's log?

Reply:
> were they emitted to their own file with no mention of the relation to the parent module in the parent module's log?

This is what was happening